regression/strings/java_char_array/test_char_array.java

```diff
@@ -0,0 +1,15 @@
+public class test_char_array {
+
+    public static void main(String[] argv)
+    {
+	String s = "abc";
+	char [] str = s.toCharArray();
+	int[] test = new int[312];
+	char c = str[2];
+	String t = argv[0];
+	char d = t.charAt(0);
+	assert(str.length == 3);
+	assert(c == 'c');
+	assert(c == d || d < 'a' || d > 'z' );
```

Comment:
Obviously just a test, but it would still be nice if the indentation were fixed.